package.json

```diff
@@ -65,7 +65,7 @@
     "metalsmith-sass": "^1.5.1",
     "metalsmith-tagcleaner": "0.0.2",
     "metalsmith-uglify": "^2.3.0",
-    "nunjucks": "^3.1.2",
+    "nunjucks": "^3.1.6",
```

Comment:
I see a bunch of commits but only Nunjucks changed?

Does this mean we could accidentally regress those others given they're only updated in the lock file?

Reply:
I don't think so – they're all dependencies of dependencies?

Reply:
Is it possible to run `npm audit` as part of CI?

If not, ignore my comments.

Reply:
It's possible, but I'd rather not make it part of this PR if that's OK. I think it's something we should look at in the new year.